### DISCUSSION TITLE
LZ deployment fails

### DISCUSSION THREAD

**Discussion by Schutzhund3** [Azure/Enterprise-Scale#1278] [Q&A]
I had deployed LZ a few times in the past for testing - but for different reasons removed it.  I *think* I may have also removed some policies in the process.  Now when I try to deploy in AzureGov I get: 
 "The element for [steps('lzSettings').denyClassicResources] could not be found in the createuidefinition" when clicking on the 'Deploy to AzureGov'.

**Comment by Springstone** [comment]
Hi @Schutzhund3, thanks for also raising this as an issue, and as you may be aware, this has been fixed, so you should hopefully be unblocked now.
